docs/docs/api/keyboard-provider.md

```diff
@@ -30,13 +30,11 @@ Could be useful to set it to `false` if you want to activate the module only on
 ## Example
 
 ```tsx
-import { KeyboardProvider } from "react-native-keyboard-controller";
+import { KeyboardProvider } from 'react-native-keyboard-controller';
 
 const App = () => {
   return (
-    <KeyboardProvider>
-      {/* other components in your tree */}
-    </KeyboardProvider>
+    <KeyboardProvider>{/* other components in your tree */}</KeyboardProvider>
```

Comment:
<img width="265" alt="image" src="https://github.com/kirillzyusko/react-native-keyboard-controller/assets/22820318/8704fea2-dea8-410d-a9de-82397e4b2442">

Looks slightly truncated 😔

Reply:
I added `The ` to enforce new lines 😄